tests/utils/LinearAlg2DTest.cpp

```diff
@@ -288,14 +287,19 @@ TEST_P(RotateAroundTest, RotateAround)
     const Point3Matrix mat = LinearAlg2D::rotateAround(origin, angle);
     const Point supposed_result = mat.apply(point);
     ASSERT_LT(vSize(supposed_result - actual_result), 2)
-        << "LinearAlg2D::rotateAround failed: Rotating " << point << " around " << origin << " for " << angle << " degrees resulted in " << supposed_result << " instead of expected " << actual_result << ".";
+      << "LinearAlg2D::rotateAround failed: Rotating " << point << " around " << origin << " for " << angle << " degrees resulted in "
+      << supposed_result << " instead of expected " << actual_result << ".";
 }
 
-INSTANTIATE_TEST_CASE_P(RotateAroundInstantiation, RotateAroundTest, testing::Values(
-    RotateAroundParameters(Point(25, 30), Point(10, 17), 90, Point(-3, 32)), //90 degrees rotation.
-    RotateAroundParameters(Point(25, 30), Point(10, 17), -90, Point(23, 2)), //-90 degrees rotation.
-    RotateAroundParameters(Point(-67, 14), Point(50, 50), 0, Point(-67, 14)), //No rotation at all.
-    RotateAroundParameters(Point(-67, 14), Point(50, 50), 12, Point(-57, -9)) //12 degree rotation. Actually ends up at [-57, -9.5]!
-));
-
-}
\ No newline at end of file
+INSTANTIATE_TEST_CASE_P(RotateAroundInstantiation,
+                        RotateAroundTest,
+                        testing::Values(RotateAroundParameters(Point(25, 30), Point(10, 17), 90, Point(-3, 32)), // 90 degrees rotation.
+                                        RotateAroundParameters(Point(25, 30), Point(10, 17), -90, Point(23, 2)), //-90 degrees rotation.
+                                        RotateAroundParameters(Point(-67, 14), Point(50, 50), 0, Point(-67, 14)), // No rotation at all.
+                                        RotateAroundParameters(Point(-67, 14),
+                                                               Point(50, 50),
+                                                               12,
+                                                               Point(-57, -9)) // 12 degree rotation. Actually ends up at [-57, -9.5]!
+                                        ));
```

Comment:
I think breaking up this line into 5 pieces makes it less readable, since it loses the alignment with the rest of the lines. What do you think?